spec/lib/deploy/activate_spec.rb

```diff
@@ -115,6 +115,13 @@
       expect(File.read(pwned_passwords_path)).to eq(pwned_passwords_content)
     end
 
+    it 'does not re-downlaod passowrds and configs if they already exist' do
```

Comment:
double typos

```suggestion
    it 'does not re-download passwords and configs if they already exist' do
```

Reply:
Also the description says passwords, but the test contents say geoip... which is the truth?